Gemfile.lock

```diff
@@ -432,69 +434,70 @@ GEM
     rack (3.1.8)
     rack-cache (1.17.0)
       rack (>= 0.4)
-    rack-protection (4.0.0)
+    rack-protection (4.1.1)
       base64 (>= 0.1.0)
+      logger (>= 1.6.0)
       rack (>= 3.0.0, < 4)
-    rack-session (2.0.0)
+    rack-session (2.1.0)
+      base64 (>= 0.1.0)
       rack (>= 3.0.0)
-    rack-test (2.1.0)
+    rack-test (2.2.0)
       rack (>= 1.3)
-    rackup (2.1.0)
+    rackup (2.2.1)
       rack (>= 3)
-      webrick (~> 1.8)
     rails-dom-testing (2.2.0)
       activesupport (>= 5.0.0)
       minitest
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.6.0)
+    rails-html-sanitizer (1.6.2)
       loofah (~> 2.21)
-      nokogiri (~> 1.14)
+      nokogiri (>= 1.15.7, != 1.16.7, != 1.16.6, != 1.16.5, != 1.16.4, != 1.16.3, != 1.16.2, != 1.16.1, != 1.16.0.rc1, != 1.16.0)
     rainbow (3.1.1)
     rake (13.2.1)
     rb-fsevent (0.11.2)
     rb-inotify (0.11.1)
       ffi (~> 1.0)
     rbtree (0.4.6)
-    rdoc (6.7.0)
+    rdoc (6.9.1)
```

Comment:
Latest rdoc version is 6.10.0, but I've downgraded to 6.9.1 since 6.10.0 makes `bundle exec rake preview_docs` fail.

https://buildkite.com/rails/docs-preview/builds/7919#01946859-4dad-43c9-808f-2ee5f2e1f19a/142-598

Reply:
It's broken by rails/sdoc#297, I think it will take some effort to update the gems so we can just keep it locked for now: #54256